index.js

```diff
@@ -0,0 +1,40 @@
+const books = [];
+const list = document.getElementById('list');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const buttonClass = document.querySelector('.buttonClass');
+
+function saveFormDataToLocalStorage(books) {
+  localStorage.setItem('books', JSON.stringify(books));
```

Comment:
-  Because you deploy your project, I test your project in your own deployment. 
- [ ] When I add a book and then refresh the page, the book doesn't remain on the page. The main idea to use local storage is exactly that. Please, check your deployment and you'll see the bug.

Reply:
Done

Reply:
Addressed this well!

Reply:
@DeliceLydia sorry Adress what?